src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,27 @@
 package core.basesyntax;
 
+import core.basesyntax.io.CsvReader;
+import core.basesyntax.io.CsvWriter;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.model.Report;
+
+import java.util.List;
+
 public class WorkWithFile {
```

Comment:
Pay attention to access modifiers. The 'CsvReader' and 'CsvWriter' are only used inside this class, so there is no need to have them as class fields with default access. Consider making them local variables within the method where they are used or if you want to reuse them, declare them as private.